src/main/java/com/exasol/containers/ContainerSynchronizationVerifier.java

```diff
@@ -0,0 +1,55 @@
+package com.exasol.containers;
+
+import static com.exasol.containers.ExasolContainerConstants.MAX_ALLOWED_CLOCK_OFFSET_IN_MILLIS;
+
+import org.testcontainers.containers.ContainerLaunchException;
+
+import com.exasol.errorreporting.ExaError;
+
+/**
+ * Check the synchronization of a given container in relation to the host running the Test Container software.
+ */
+public class ContainerSynchronizationVerifier {
+    private final ContainerTimeService timeService;
+
+    /**
+     * Create a new {@link ContainerSynchronizationVerifier}.
+     *
+     * @param timeService service that provide the time in the container
+     * @return new {@link ContainerSynchronizationVerifier} instance
+     */
+    public static ContainerSynchronizationVerifier create(final ContainerTimeService timeService) {
```

Comment:
To simplify usage: get a `Container` as argument and create the `ContainerTimeService` here.

Reply:
That breaks the abstraction in my eyes. At the moment the Verifier does not need to know what a container is, only that the time service exists and that it needs one.